### PR TITLE
Add dark colors to the theme

### LINKED ISSUE #49: Add darker color to the color palette

### PR DESCRIPTION
**Pull request by hakimelek**
This change removes tints colors from the theme and replace them with 3 versions of the color:
- brighten version of the color
- actual theme color
- darker version of the color

resolves #49

### CODE REVIEW

File: src/theme.js [hunk 1]
```diff
@@ -76,18 +76,12 @@ const flatten = (name, colors) => colors
...
-  0.6,
-  0.8
-]
-const blues = tints.map(tint(blue))
-const grays = tints.map(tint(gray))
-const greens = tints.map(tint(green))
-const reds = tints.map(tint(red))
-const oranges = tints.map(tint(orange))
-const purples = tints.map(tint(purple))
+const blues = [brighten(blue), blue, darken(blue)]
```
Comment (jxnblk) [inline src/theme.js]: It sounded like @oliverdumoulin wasn't using a color function for his updated color scale anymore, so I think we'll want to remove the chroma-js dependency and just hard code the values in

Comment (hakimelek) [reply src/theme.js]: @jxnblk We are depending on the tint function (depending on chroma) in the legacy theme. Should we hard code those values as well?

Comment (jxnblk) [reply src/theme.js]: Yeah, I think that's fine – probably worth getting rid of that dependency if we can

**Comment by hakimelek** [comment]
Closing this in favor of #79.

**Pull request closed without merging**